### PR TITLE
feat(core,platform): update angular 15 and nx to v15

### PR DESCRIPTION
**Pull request by N1XUS**
**BREAKING CHANGE:**

* Updated @angular library to v15;
* Updated nx library to v15;
* Updated other third-party libraries to latest version;
* Applied necessary migrations;
* Adjusted unit tests;
* For projects without unit tests angular failed karma execution with error that no tests were found, so filler test (dummy) was added to such projects;
* Build target now ES2022

### CODE REVIEW

**Review by platon-rov — APPROVED** [review]
I don't see any major changes, approving.

Please fix pipeline.

File: migrations.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
+{
```
[the remark this answers is not in the document]
Comment (N1XUS) [reply migrations.json:1]: @platon-rov will be removed after all new libs are added to this branch

File: libs/core/schematics/migrations/migration-0-37/index.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ function removeStylesFromConfig(): Rule {
                     ];
 
                     const stylesToRemove = [
-                        'node_modules/fundamental-styles/dist/icon.css',
+                        '~fundamental-styles/dist/icon.css',
```
Comment (g-cheishvili) [inline libs/core/schematics/migrations/migration-0-37/index.ts:35]: What happened here?

**Review by g-cheishvili — APPROVED** [review]

File: libs/core/schematics/migrations/migration-0-37/index.ts (hunk 2 above)
Comment (N1XUS) [reply libs/core/schematics/migrations/migration-0-37/index.ts]: was playing with paths, accidentally affected this file

**Pull request merged**